src/frontend/src/pages/TeamsPage/TeamMembersPageBlock.tsx

```diff
@@ -41,13 +43,22 @@ const TeamMembersPageBlock: React.FC<TeamMembersPageBlockProps> = ({ team }) =>
 
   if (allUsersIsError) return <ErrorPage message={allUsersError?.message} />;
   if (allUsersIsLoading || setTeamMembersIsLoading || setTeamHeadIsLoading || !users) return <LoadingIndicator />;
+  let hasPerms = false;
+
+  if (auth.user) {
```

Comment:
Please define your perms value as a `const` so you don't have to mutate it using conditionals - just define it as the different options separated by `||` like how the original `hasPerms` was defined